scripts/changelog.ts

```diff
@@ -0,0 +1,386 @@
+/*
```

Comment:
What's the purpose of those scripts?

Reply:
This is needed to generate the release notes @szczepaniak-michal